### PR TITLE
#2 Implemented migrations for different tables.

### PR DESCRIPTION
**Pull request by nkovalenko-sc**
Implements #2.

### CODE REVIEW

File: src/api/inheritanceSettings.js [hunk 1]
```diff
@@ -62,12 +62,12 @@ class InheritanceSettingsDao {
     const transaction = await this.db.transaction();
```
Comment (shirleyqiantingliu) [inline src/api/inheritanceSettings.js]: Upper lines, should return constant error_code

File: src/db/models/InheritanceSettings.js [hunk 2]
```diff
@@ -39,7 +37,7 @@ const init = db => {
       category: {
         field: 'Category',
         type: Sequelize.ENUM,
```
Comment (shirleyqiantingliu) [inline src/db/models/InheritanceSettings.js]: should it be type: Sequelize.STRING(50)

Comment (nkovalenko-sc) [reply src/db/models/InheritanceSettings.js]: yes, it's already fixed.

File: src/api/inheritanceSettings.js (hunk 1 above)
Comment (nkovalenko-sc) [reply src/api/inheritanceSettings.js]: Not sure it should be like that. It's message for and API user, but not for the real client.

File: src/db/index.js [hunk 3]
```diff
@@ -1,30 +1,47 @@
...
-        console.log(migration);
-      }
+    storageOptions: {
+      sequelize: db,
+      model: migrationsModel
     },
     migrations: {
       pattern: /^\d{14}-\d+\.\w+\.js$/,
-      params: [db],
+      params: [db, tableName],
```
Comment (alevchuk-sc) [inline src/db/index.js]: Pattern is still the same.

File: src/db/migrations/20190402170338-2353.main.js [hunk 4]
```diff
@@ -3,6 +3,6 @@
 const createInheritanceSettings = require('./inheritance-settings/createInheritanceSettings');
 
 module.exports = {
-  up: async db => createInheritanceSettings(db.queryInterface),
-  down: db => db.queryInterface.dropTable('InheritanceSettings')
+  up: async (db, tableName) => createInheritanceSettings(db.queryInterface, tableName),
```
Comment (alevchuk-sc) [inline src/db/migrations/20190402170338-2353.main.js]: Lets maybe not pass tableName as parameter but rather config object `{ serviceName: 'invoice' }`?

File: src/api/inheritanceSettings.js [hunk 5]
```diff
@@ -45,8 +45,8 @@ class InheritanceSettingsDao {
   }
 
 
-  findAllByCustomerId(customerId) {
-    return this.db.models.InheritanceSettings.findAll({ where: { customerId } });
+  findAll(params) {
+    return this.db.models.InheritanceSettings.findAll({ where: params });
```
Comment (alevchuk-sc) [inline src/api/inheritanceSettings.js]: I think we shouldn't expose actual sequelize instances cause it breaks the whole concept of api.

**Pull request merged**